src/Components/Scenes/Hydration/NewJob/NewJob/NewJob.js

```diff
@@ -12,8 +14,10 @@ const chromeStyles = makeStyles(theme => ({
   },
   // Placeholder pattern until canvas is implemented
   canvas: {
-    background:
-      'repeating-linear-gradient(45deg, #EDF5FF, #EDF5FF 15px, #FFFFFF 5px, #FFFFFF 20px)',
+    backgroundColor: color.g50,
+    backgroundSize: '96px 96px, 96px 96px, 16px 16px, 16px 16px',
+    backgroundPosition: '0 0, 0 0, 0 0, 0 0',
+    backgroundImage: `linear-gradient(${lighten(color.g100, 0.4)} 1px, transparent 1px), linear-gradient(90deg, ${lighten(color.g100, 0.4)} 1px, transparent 1px), linear-gradient(${lighten(color.g100, 0.4)} 1px, transparent 1px), linear-gradient(90deg, ${lighten(color.g100, 0.4)} 1px, transparent 1px)`,
```

Comment:
Small nitpick, but maybe we could do 
```
`linear-gradient(${lighten(color.g100, 0.4)} 1px, transparent 1px), `
+ `linear-gradient(90deg, ${lighten(color.g100, 0.4)} 1px, transparent 1px), `
+ `linear-gradient(${lighten(color.g100, 0.4)} 1px, transparent 1px), `
+ `linear-gradient(90deg, ${lighten(color.g100, 0.4)} 1px, transparent 1px)`
```
Since that line is reallll long. Prettier might just boot it back to this though, so meh

Reply:
Good call! Updated